{"uuid": "7d0a0c52-219b-415b-9364-af7157e71a7b", "children": ["6d8b8c46-9f29-471d-9203-387f27c4b6b8"], "befores": [{"name": "web_driver_init", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'set_environment' referenced before assignment\n", "trace": "  File \"C:\\Users\\Artem Kozlov\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Artem Kozlov\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Artem Kozlov\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Artem Kozlov\\PycharmProjects\\POM_Selenium_Pytest_Task1\\tests\\conftest.py\", line 86, in web_driver_init\n    allure_environment_status, set_environment = set_allure_environment_properties(browser_name, allure_dir, driver)\n  File \"C:\\Users\\Artem Kozlov\\PycharmProjects\\POM_Selenium_Pytest_Task1\\tests\\conftest.py\", line 18, in set_allure_environment_properties\n    if not set_environment:\n"}, "start": 1642068797679, "stop": 1642068800201}], "start": 1642068797679, "stop": 1642068800316}